disabledTestParser/issue_status.py

```diff
@@ -0,0 +1,40 @@
+import json
+import requests
+
+
+def json_parser(json_obj):
+    jsonLoad = json.loads(json_obj)
+    return jsonLoad["GIT_ISSUE"]
+
+def find_state(query_url):
+    params = {'accept': 'application/vnd.github.v3+json',
+              'state': 'all'}
+    try:
+        response = requests.get(query_url, params=params).json()
+        json_str = json.dumps(response)
+        return json.loads(json_str)["state"]
+    except:
+        print("An exception occurred! Url is invalid")
+
+def main():
+    base_url = f'https://api.github.com/repos/'
+    f = open('issue.json',)
```

Comment:
A minor issue, could you rename `issue.json` to `ProblemList.json` to match with https://github.com/adoptium/aqa-tests/pull/2858? Thanks.